symphony/content/content.blueprintsdatasources.php

```diff
@@ -181,6 +181,11 @@ public function __form()
                         } else {
                             $fields['static_xml'] = trim($existing->grab());
                         }
+                        $fields['static_xml'] = htmlspecialchars(
+                            $fields['static_xml'],
+                            ENT_XML1|ENT_COMPAT,
+                            'UTF-8'
+                        );
```

Comment:
I would personally move that assignment to line 949, because that where we need it (?)

Reply:
And maybe dropping the `General::sanitize` call is needed to fix the issue raised by @michael-e ?

Reply:
Unfortunately, removing the call to `General::sanitize` (when creating the `Widget::Textarea`) does not help at all with my issue.

Reply:
:(

Reply:
> Unfortunately, removing the call to General::sanitize (when creating the Widget::Textarea) does not help at all with my issue.

I think it does?